features/bevragen/persoon/verblijfplaats/protocollering-vraag.feature

```diff
@@ -0,0 +1,67 @@
+#language: nl
+
+Functionaliteit: protocollering van de vraag voor verblijfplaats
+
+  Rule: Met fields gevraagde velden worden geprotocolleerd als de elementnummers volgens Logisch ontwerp BRP
+    Dit is een 6-cijferige code, met zo nodig voorloopnul voor categorieën.
+    Ook alle velden die worden meegeleverd voor het bepalen van het verblijfplaats type worden opgenomen.
```

Comment:
`Ook alle velden die worden meegeleverd voor het bepalen van het verblijfplaats type worden vastgelegd in het veld 'request_zoek_rubrieken'.`

Reply:
aangepast